src/app/tests/suites/TestCluster.yaml

```diff
@@ -626,6 +626,72 @@ tests:
       arguments:
           value: ""
 
+    # Tests for Char String attribute
+
+    - label: "Read attribute CHAR_STRING Default Value"
+      command: "readAttribute"
+      attribute: "char_string"
+      response:
+          value: ""
+
+    - label: "Write attribute CHAR_STRING"
+      command: "writeAttribute"
+      attribute: "char_string"
+      arguments:
+          value: "☉Test☉"
+
+    - label: "Read attribute CHAR_STRING"
+      command: "readAttribute"
+      attribute: "char_string"
+      disabled: true
+      response:
+          value: "☉Test☉"
+
+    - label: "Write attribute CHAR_STRING"
+      command: "writeAttribute"
+      attribute: "char_string"
+      arguments:
+          value: "☉TestValueLongerThan10☉"
+
+    - label: "Read attribute CHAR_STRING"
+      command: "readAttribute"
+      attribute: "char_string"
+      disabled: true
+      response:
+          value: "☉TestVal"
```

Comment:
This is pretty surprising to me.  I would have expected a too-long write to fail entirely, not just write whatever fits...

If the spec does not define behavior clearly here, please file a spec issue.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/3558